problems/arraysIntro.js

```diff
@@ -64,29 +78,47 @@ function arrayToString() {}
  * @param {Array} array2
  * @returns {string|Array}
  */
-function largerArray() {}
+function largerArray(array1, array2) {
+      if (array1.length > array2.length){
+        return array1
+      }else if (array2.length === array1.length){
+        return "They are the same size"
+      }else{
+          return array2
+      }
 
+}
 /**
  * Takes in an array and returns the middle element.
  * If the array is even in length return the middle two elements in their own array.
  * @param {Array} array
  * @returns {*|Array}
- */
-function middleElement() {}
-
+ */ 
+ function middleElement(array) {
+   let num = Math.floor(array.length / 2) //this get you to the halfway point
```

Comment:
This looks great! I'd recommend using the variable name idx instead of num because it represents a future index.